keylime/registrar_common.py

```diff
@@ -281,11 +281,16 @@ def do_POST(self):
                 return
 
             # try to encrypt the AIK
-            (blob, key) = initialize_tpm.encryptAIK(
+            aik_enc = initialize_tpm.encryptAIK(
                 agent_id,
                 base64.b64decode(ek_tpm),
                 base64.b64decode(aik_tpm),
             )
+            if aik_enc is not None:
+                (blob, key) = aik_enc
+            else:
+                logger.warning("Agent %s failed encrypting AIK", agent_id)
```

Comment:
Before returning you should still send a response. E.g.:
```
web_util.echo_json_response(self, 400, f"Error: failed encrypting AK")
```

Reply:
668a1bd